### PR TITLE
make ATen/native/cuda/RreluWithNoise.cu

### PR DESCRIPTION
**Pull request by dagitses**
make ATen/native/cuda/RreluWithNoise.cu

Test Plan: Rely on CI.

### CODE REVIEW

File: aten/src/ATen/native/cuda/RreluWithNoise.cu [hunk 1]
```diff
@@ -95,9 +95,9 @@ inline void _rrelu_with_noise_cuda_train(
     rng_engine_inputs = gen->philox_cuda_state(counter_offset);
   }
 
-  scalar_t* input_data = input.data_ptr<scalar_t>();
-  scalar_t* noise_data = noise.data_ptr<scalar_t>();
-  scalar_t* output_data = tmp_output.data_ptr<scalar_t>();
+  const scalar_t* input_data = input.const_data_ptr<scalar_t>();
+  scalar_t* noise_data = noise.mutable_data_ptr<scalar_t>();
```
Comment (ezyang) [inline aten/src/ATen/native/cuda/RreluWithNoise.cu:99]: This looks extremely suspect

Comment (dagitses) [reply aten/src/ATen/native/cuda/RreluWithNoise.cu:99]: take a look at line 63 though.

**Review by ezyang — APPROVED** [review]

**Pull request closed without merging**